core/slice.go

```diff
@@ -296,7 +296,7 @@ func (sl *Slice) Append(header *types.Header, domPendingHeader *types.Header, do
 		if order < nodeCtx {
 			// Store the inbound etxs for dom blocks that did not get picked and use
 			// it in the future if dom switch happens
-			rawdb.WriteInboundEtxs(sl.sliceDb, block.Hash(), newInboundEtxs.FilterToLocation(common.NodeLocation))
```

Comment:
Is the assumption that dom already filtered this list?

Reply:
Yes, Dom filters is and sends it down, nonneed to filter twice in my opinion, its very expensive